app/models/pipeline_run_stage.rb

```diff
@@ -296,7 +308,7 @@ def db_load_alignment
     pr.updated_at = Time.now.utc
     pr.save
     # aggregate the data at genus level
-    pr.generate_aggregate_counts('genus')
+    pr.generate_aggregate_counts('genus') unless pr.pipeline_branch == 'charles/hit-calling'
```

Comment:
yeah this too needs to be based on something else, pipeline version perhaps

Reply:
same comment as for the other occurrence